brevent/src/main/res/values-es-v26/strings.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="brevent_about_developer_adb">No desconecte \"Depuración por USB\", no cambie el \"configuratión de USB\"</string>
+    <string name="brevent_about_developer_adb">No desconecte \"Depuración por USB\", no cambie la \"configuración USB\"</string>
```

Comment:
https://github.com/aosp-mirror/platform_frameworks_base/blob/master/packages/SettingsLib/res/values-es/strings.xml#L215

Reply:
I can't repair this, I can't edit this string. I don't know how is the problem...